cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/adapters/RawApplicationLog.java

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
I'm not sure whether we have this copyright (since this is a fork). Remove this comment.